nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/test/java/org/apache/nifi/processors/mongodb/GetMongoIT.java

```diff
@@ -67,20 +69,23 @@
 
     @Before
     public void setup() {
-        runner = TestRunners.newTestRunner(GetMongo.class);
+        runner = TestRunners.newTestRunner(TestGetMongo.class);
         runner.setVariable("uri", MONGO_URI);
         runner.setVariable("db", DB_NAME);
         runner.setVariable("collection", COLLECTION_NAME);
         runner.setProperty(AbstractMongoProcessor.URI, "${uri}");
         runner.setProperty(AbstractMongoProcessor.DATABASE_NAME, "${db}");
         runner.setProperty(AbstractMongoProcessor.COLLECTION_NAME, "${collection}");
-        runner.setProperty(GetMongo.USE_PRETTY_PRINTING, GetMongo.YES_PP);
+        runner.setProperty(GetMongo.USE_PRETTY_PRINTING, GetMongo.GM_TRUE);
         runner.setIncomingConnection(false);
+        runner.setValidateExpressionUsage(true);
```

Comment:
I believe that's the default here, probably best to not have this line

Reply:
Removed.